src/pymor/operators/basic.py

```diff
@@ -133,12 +134,28 @@ def apply_inverse_transpose(self, U, mu=None, least_squares=False):
             return transpose_op.apply_inverse(U, mu=mu, least_squares=least_squares)
 
     def as_range_array(self, mu=None):
-        assert isinstance(self.source, NumpyVectorSpace)
-        return self.apply(self.source.make_array(np.eye(self.source.dim)), mu=mu)
```

Comment:
Instead of the special case for `BlockVectorArray`, we should be able to write:
```
return self.apply(self.source.from_data(np.eye(self.source.dim)), mu=mu)
```